private/dir.go

```diff
@@ -0,0 +1,55 @@
+package private
```

Comment:
There is `doc.go`.

`private` is simply a grab-bag of stuff which is useful to share between transports, primarily for policy configuration scoping (Docker + OpenShift share Docker Reference formatting, dir + OCI share path canonicalization).